docs/ir/ir-breaking-changes.rst

```diff
@@ -112,8 +112,48 @@ This causes differences in some contracts, for example:
          }
      }
 
-Previously `f()` would return `0x6465616462656566313564656164000000000000000000000000000000000010` (it has correct length, and correct first 8 elements, but than it contains dirty data which was set via assembly).
-Now it is returning `0x6465616462656566000000000000000000000000000000000000000000000010` (it has correct length, and correct elements, but doesn't contain dirty data).
+Previously ``f()`` would return ``0x6465616462656566313564656164000000000000000000000000000000000010`` (it has correct length, and correct first 8 elements, but than it contains dirty data which was set via assembly).
```

Comment:
```suggestion
Previously ``f()`` would return ``0x6465616462656566313564656164000000000000000000000000000000000010`` (it has correct length, and correct first 8 elements, but then it contains dirty data which was set via assembly).
```

Reply:
I'd also say that it's not the data that is dirty but the positions in the slot :)

Reply:
Does `superfluous` data make sense?

Reply:
Yeah, though "dirty bits" seem like the more typical way to refer to this. "Dirty data" just sounds as if there's something wrong with the value itself rather than its location.

Reply:
Perhaps adding a brief definition of "dirty data" -> superfluous bits that may be set although they shouldn't technically be earlier in the docs be a nice way to introduce the reader to what we mean by "dirty data"?

Reply:
I'm not sure it's necessary. My issue here is not that it's hard to understand (I think anyone who understands "dirty bits"/"superfluous data" will get it), just that this way to refer to it sounds a little unusual to me.

But it's also just a small nitpick. If you really think "dirty data" is the best way to refer to it, I'd leave it as is.